_pages/how-to.md

```diff
@@ -2,32 +2,188 @@
 layout: "q-and-a"
 title: How to...
 ---
+
 # {{ page.title }}
 
-Hello, visitor! Our internal page has a lot of tips from fellow 18F staffers
-about things they really wished they had known them when they started. These
-tips fall under the general categories of:
-
-* You're a fed now!
-* Jargon and acronyms
-* Travel, events, and media
-* Training
-* Leave
-* Common work tasks
-* Supplies
-* Software
-* Phones
-* Email
-* Information Technology
-* Passwords and authentication
-* GitHub
-* Google Apps and Tools
-* Legal and ethics questions
-* Freedom of Information Act
-* Federal records
-* Privacy
-* DevOps
-* Email groups and calendars
-* Washington D.C.
-* New York City
-* Human Resources
+These tips come from fellow 18F staffers who really wished they had known them when they started. Feel free to ask for help with any of these on Slack. If something doesn’t make sense, ask in the #general channel and someone will happily lend a hand. Also, **please help the next person by improving this documentation as you go**. You can [file an issue](https://github.com/18F/hub/issues), click the "edit this page" link at the bottom of this (and all) pages, or branch-add-pull request from [the Hub repo](https://github.com/18F/hub).
+
+## You're a fed now
+
+* Here are [useful GSA links]({{ site.baseurl }}/gsa-links/), all in one place.
+
+* Find out more about the [GSA Federal Credit Union](../private/fcu/) and its potentially lifelong benefits.
```

Comment:
I'll follow up with a PR (unless someone beats me to it) to wrap the `../private` links in a `{% unless site.public %}` block.